recipes/hidefix/meta.yaml

```diff
@@ -39,7 +39,8 @@ requirements:
     - numpy
     - xarray
     - netCDF4
-    - ucrt  # [win]
+    - ucrt          # [win]
+    - vc14_runtime  # [win]
```

Comment:
I'd try one and then the other. Both at the same time you may never know which one is really needed and/or why it is still broken.

Reply:
Yes. Only ucrt fixed the ucrt error. I'm trying with the compiler cxx template on win as well.